internal/controller/instaslice_daemonset.go

```diff
@@ -365,7 +365,7 @@ func (r *InstaSliceDaemonsetReconciler) Reconcile(ctx context.Context, req ctrl.
 					return ctrl.Result{RequeueAfter: 1 * time.Second}, nil
 				}
 				nodeName := os.Getenv("NODE_NAME")
-				if errUpdatingNodeCapacity := r.updateNodeCapacity(ctx, nodeName, allocations.Profile, emulatorMode, allocations.Allocationstatus); errUpdatingNodeCapacity != nil {
+				if errUpdatingNodeCapacity := r.updateNodeCapacity(ctx, nodeName, allocations, emulatorMode, allocations.Allocationstatus); errUpdatingNodeCapacity != nil {
```

Comment:
We don't need to have `allocations.Allocationstatus` as a one more argument as we already have `allocations`

Reply:
Thanks, done!